formats/developers/filter-and-filterset.txt

```diff
@@ -163,40 +188,44 @@ structure of the resulting xml.
 
 ::
 
-    <ome:FilterSet ID="FilterSet:1" Manufacturer="Ink Inc." Model="Mk 3" LotNumber="K753">
-        <ome:ExcitationFilterRef ID="Filter:1"/>
-        <ome:ExcitationFilterRef ID="Filter:2"/>
-        <ome:DichroicRef ID="Dichroic:1"/>
-        <ome:EmissionFilterRef ID="Filter:3"/>
-        <ome:EmissionFilterRef ID="Filter:4"/>
-    </ome:FilterSet>
-    <ome:Filter ID="Filter:1" Manufacturer="Ink Inc." Model="Medium 490" LotNumber="J23"
-        Type="BandPass" FilterWheel="Disk 7">
-        <ome:TransmittanceRange Transmittance="0.80" CutIn="450" CutOut="530"/>
-    </ome:Filter>
-    <ome:Filter ID="Filter:2" Manufacturer="Ink Inc." Model="Medium 520" LotNumber="J34"
-        Type="BandPass" FilterWheel="Disk 7">
-        <ome:TransmittanceRange Transmittance="0.75" CutIn="500" CutOut="570"/>
-    </ome:Filter>
-    <ome:Filter ID="Filter:3" Manufacturer="Ink Inc." Model="Medium 580" LotNumber="J12"
-        Type="BandPass" FilterWheel="Disk 7">
-        <ome:TransmittanceRange Transmittance="0.85" CutIn="550" CutOut="620"/>
-    </ome:Filter>
-    <ome:Filter ID="Filter:4" Manufacturer="Ink Inc." Model="Medium 630" LotNumber="J09"
-        Type="BandPass" FilterWheel="Disk 7">
-        <ome:TransmittanceRange Transmittance="0.90" CutIn="590" CutOut="680"/>
-    </ome:Filter>
-    <ome:Filter ID="Filter:5" Manufacturer="Ink Inc." Model="Medium 630" LotNumber="J29"
-        Type="BandPass" FilterWheel="Disk 7">
-        <ome:TransmittanceRange Transmittance="0.60" CutIn="630" CutOut="690"/>
-    </ome:Filter>
-    <ome:Filter ID="Filter:6" Model="MirrorBlock Mk II" LotNumber="M538" Type="Dichroic"/>
-    <ome:Dichroic ID="Dichroic:1" Model="HFT 405/488/543/633"/>
-    <ome:Dichroic ID="Dichroic:2" Model="MirrorBlock MK II" LotNumber="M539"/>
-
-
-
-This second example shows the Image side - the LightPath structure of the 
+    <FilterSet ID="FilterSet:1" Manufacturer="Ink Inc." Model="Mk 3" LotNumber="K753">
+    <ExcitationFilterRef ID="Filter:1"/>
+    <ExcitationFilterRef ID="Filter:2"/>
+    <ExcitationFilterRef ID="Filter:3"/>
+    <ExcitationFilterRef ID="Filter:4"/>
+    <DichroicRef ID="Dichroic:1"/>
+    <EmissionFilterRef ID="Filter:5"/>
+    <EmissionFilterRef ID="Filter:6"/>
+    </FilterSet>
+    <FilterSet ID="FilterSet:2" Manufacturer="Ink Inc." Model="Mk 3" LotNumber="K753"/>
```

Comment:
Ok, well the example has been updated to reflect the actual sample file -http://downloads.openmicroscopy.org/images/OME-XML/2015-01/filter.ome.xml so if that sample makes no sense, you need to talk to @sbesson about how to fix it, because I just pasted the content in this PR from that sample.